src/generators/index.ts

```
@@ -31,6 +31,22 @@ export async function generateDocumentation(
         const spinner = ora(`Creating ${path.basename(file.path)}...`).start();
         try {
           await fs.ensureDir(path.dirname(file.path));
+          
```

ESLint

[error] Delete `··········`

(prettier/prettier)